lib/ExampleNameItem.dart

```diff
@@ -33,18 +32,15 @@ class ExampleNameItem extends StatelessWidget {
         child: Container(
           decoration: BoxDecoration(
             border: Border(
-              left: BorderSide(
-                width: 4.0,
-                color: Theme
-                    .of(context)
-                    .accentColor,
-              ),
+              left:
+                  BorderSide(width: 4.0, color: Theme.of(context).accentColor),
             ),
```

Comment:
I do not know if we really needed this change of the formatting, as it was already formatter as per the Dart guidelines.